reference/CloudPay-API-Specification.yaml

```diff
@@ -2235,6 +2235,10 @@ components:
             type: string
             minLength: 1
             readOnly: true
+          Status:
+            type: string
+            enum: ['ACTIVE', 'CANCELLED']
```

Comment:
i assume UserSummary won't be pulling out expired subscriptions so active & cancelled should do right?